lib/utils/chainPromise.js

```diff
@@ -18,7 +17,7 @@ let currentPromise = Promise.resolve();
  */
 
 const chainPromise = callback => (...args) => {
-	const wrappedCallback = stackTraceWrapper(() => callback(...args));
```

Comment:
this was here so that commands (those in commands/ dir) have correct error stack, does it work without it ?

Reply:
@givehug could you do me a favor and check this? I don't remember much of this anymore.